app/utils/AddASettlorViewHelper.scala

```diff
@@ -72,6 +72,12 @@ class AddASettlorViewHelper(userAnswers: UserAnswers, draftId: String)(implicit
         controllers.routes.FeatureNotAvailableController.onPageLoad().url,
         removeUrl = routes.RemoveSettlorController.onPageLoad(index, draftId).url
       )
+      case SettlorBusinessTypeViewModel(_, name, _) => AddRow(
+        name,
+        messages("entity.settlor.business"),
+        controllers.routes.FeatureNotAvailableController.onPageLoad().url,
```

Comment:
I think we can support this feature. However, the 'In Progress' functionality adds complexity, since ideally we'd like to redirect them to the check your answers, but the answers may not be in a submittable state (i.e. the entity is in progress). The way I see it we have 4 options:

1. Redirect them to the first page in the journey by default
2. Redirect them to the check your answers page, but have logic that backtracks from the end of the journey to the first unanswered page it can find. This becomes a bit of a faff for a non-linear journey, but there is an implementation of it in register-trust-asset-frontend for a PropertyOrLand asset (this implementation is very specific to PropertyOrLandAsset and could possibly be refactored to make it more universal) 
3. Use the status value to send them to check your answers if the status is Complete, or the first page in the journey if the status is InProgress
4. Leave it as FeatureUnavailable

cc @adamconder

Reply:
It's currently Feature unavailable for all change journeys for setllors so I think that belongs in a separate ticket. Thoughts @adamconder ?

Reply:
@BriWak You're correct this doesn't exist for the existing journeys but I'd like to implement it if we could. I agree, the in progress does cause a problem.

I quite like option 3 as it's relatively simple just to make a switch between two. Or option 1 forces everyone through the same journey, like they would see in maintaining.
It's more a product/UX question but for now I'd go for the first question in the series.

Reply:
Done now, and I've done it for the other settlor types as well